### ISSUE TITLE
How to use MiniBlog

### ISSUE THREAD

**Issue by phatcher** [madskristensen/MiniBlog#237]
Sorry to be dumb, but what's the best way to consume the source., this is my idea at the moment...

1. Make a fork
2, Make two branches from master, dev and publish

Then on "publish" I have my custom configuration changes and posts and I push that branch to azure.

On "dev" I can do code development and then merge to publish or to master if I want to issue a pull-request.

**Comment by mieliespoor** [comment]
Yep, that's pretty much how it would work. This way would also allow you to first pull in changes from other repo's and test it before publishing to your blog.

**Comment by phatcher** [comment]
Also, is this intended to be a "real" project or a bit of a technology sampler?

Would be nice to add more themes, Disqus comment support, Google Analytics etc - pull requests ok?

**Comment by RandomlyKnighted** [comment]
Yes, it's a real project. There are tons of people who use Miniblog as their blog engine.

**Comment by phatcher** [comment]
But then Mads just rejected my GA and copyright patch 'cos it's not Core - which means that the features will never improve and I have to develop everything I want myself

**Comment by JohanPetersson** [comment]
Adding Disqus and GA takes seconds, why just add that bloat to the core project? Just develop your own theme and use that instead. I have a fork with my own theme.

**Comment by phatcher** [comment]
Ok, that might be the way to go - covers off the core layout issues such as the GA tag etc

**Comment by jonaslorander** [comment]
I've seen several forks with both GA and Disqus support. There is even one with some pretty advanced features for adding your own commenting system with ease. Browse the forks and you just might find what you are looking for :)

**Comment by phatcher** [comment]
That's what I've been doing :-)
